NAMESPACE

```diff
@@ -143,7 +142,6 @@ export(format_eoxxstt_default)
 export(format_reason_default)
 export(get_admiral_option)
 export(get_duplicates_dataset)
-export(get_hori_data)
```

Comment:
Was the removal of these exports intentional?

Reply:
Yes, `get_hori_data()` is a helper function for `derive_param_computed()`. It is unlikely that users want to call it directly.

Reply:
Makes sense, should we create a separate issue to clean up other helper-functions/migrate them over to `admiraldev` ? For example, something like `assert_valid_queries()` ?

Reply:
Gotcha I hadn't realized we had a single-use-case rule/exception, also makes sense